library/proc_macro/Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.0.0"
 edition = "2021"
 
 [dependencies]
+literal-escaper = { path = "../../src/literal-escaper" }
```

Comment:
I think `literal-escaper` should be under `library/` so it gets included in the `rust-src` component -- then it will be available for `rust-analyzer`, `cargo -Zbuild-std`, etc.

Reply:
Good idea!

Reply:
When moving it to `library` I get:

```
WARNING: The `change-id` is missing in the `config.toml`. This means that you will not be able to track the major changes made to the bootstrap configurations.
NOTE: to silence this warning, add `change-id = 135832` at the top of `config.toml`

thread 'main' panicked at src/bootstrap/src/core/metadata.rs:66:13:
multiple crates with the same path: literal-escaper
```